org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/contentassist/CompletionProposalReplacementProvider.java

```diff
@@ -408,6 +408,9 @@ private void appendReplacementString(StringBuilder buffer, CompletionProposal pr
 			String str = proposal.getKind() == CompletionProposal.TYPE_REF ? computeJavaTypeReplacementString(proposal) : String.valueOf(proposal.getCompletion());
 			if (client.isCompletionSnippetsSupported()) {
 				str = CompletionUtils.sanitizeCompletion(str);
+				if (proposal.getKind() == CompletionProposal.PACKAGE_REF && str != null && str.endsWith(".*;")) {
+					str = str.replace(".*;", ".${0:*};");
```

Comment:
you need to check if the client supports snippets

Reply:
See line 409

Reply:
ok my bad